core/src/main/java/tc/oc/pgm/events/MapPoolAdjustEvent.java

```diff
@@ -0,0 +1,63 @@
+package tc.oc.pgm.events;
+
+import javax.annotation.Nullable;
+import org.bukkit.command.CommandSender;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.rotation.MapPool;
+
+/** MapPoolAdjustEvent is called when the active {@link MapPool} is set to another * */
+public class MapPoolAdjustEvent extends Event {
```

Comment:
Not sure this event is necessary. Since you're just printing to chat, I would skip the event and just print.

Reply:
So personally I plan on having a plugin which is going to listen to this event, to be used in order to perform other cool stuff. As I wanted to keep specific features out of PGM for the sake of not evading the scope. Is that alright or would you still like it removed?

Reply:
Hmm, I'm okay with keeping this for now, but there's no guarantees it will still exist when the API is published and 1.0 is released. So you can keep it for now.